postmortems/005 - Postmortem for faulty Auth0 Webhook.md

```diff
@@ -0,0 +1,35 @@
+# 004 - Postmortem for faulty Auth0 PostLogin JS webhook
+
+## Level: Internal
+
+## Author(s)
+
+- Luis Fernando De Pombo <l@iasql.com>
+
+## Summary
+
+The addition of an Auth0 PostLogin JS webhook, which turned out to run during log-in and not after it, caused a redirect loop in the production dashboard and made production unavailable when the webhook failed to run to completion which would happen most of the time.
+
+## Timeline
+
+- **2023-03-14**: @depombo added the faulty Auth0 PostLogin JS webhook
+- **2023-03-15**: @depombo tried to access the production dashboard and noticed that it wasn't possible
+
+## Detection
+
+The problem was around for roughly 16 hours when the faulty Auth0 PostLogin JS webhook was added until there was an attempt to access the production dashboard that did not work.
```

Comment:
It would probably be good to note that it was our own usage of production here, and that's why this can be classified as an "Internal" outage.